### DISCUSSION TITLE
No Windows Updates

### DISCUSSION THREAD

**Discussion by pdifeo** [amidaware/tacticalrmm#1994] [Q&A] [answered]
Why doesn't the console show me that there are no patches to install while the computer tells me there are?

![image](https://github.com/user-attachments/assets/97e5a7a5-3723-4950-92be-ff3bcf074e1a)

![image](https://github.com/user-attachments/assets/93f6b8cb-ffd3-497a-a999-fec4834d9858)

**Comment by wh1te909** [comment, marked as answer]
https://learn.microsoft.com/en-us/windows/release-health/resolved-issues-windows-11-23h2#3351msgdesc

you'll need to manually install that update and issue will be resolved

**Reply by pdifeo** [reply]
It's work. Thanks
